sdk/storage/Azure.Storage.Blobs/src/BlobClientBuilderExtensions.cs

```diff
@@ -56,15 +56,6 @@ public static IAzureClientBuilder<BlobServiceClient, BlobClientOptions> AddBlobS
             return builder.RegisterClientFactory<BlobServiceClient, BlobClientOptions>((options, token) => new BlobServiceClient(serviceUri, token, options));
         }
 
-        /// <summary>
-        /// Registers a <see cref="BlobServiceClient"/> instance with the provided <paramref name="serviceUri"/> and <paramref name="tokenCredential"/>
-        /// </summary>
-        public static IAzureClientBuilder<BlobServiceClient, BlobClientOptions> AddBlobServiceClientWithTokenCredential<TBuilder>(this TBuilder builder, Uri serviceUri, TokenCredential tokenCredential)
```

Comment:
I don't know that you can remove this, as it'll be a breaking change.   You'll probably want to mark as obsolete and editor non-browsable unless KC or Chris disagrees.

Reply:
D'oh!  My bad.